Gruntfile.js

```diff
@@ -322,13 +318,6 @@ module.exports = function(grunt) {
     }
   });
 
-  // global beforeEach task
-  if (!process.env.TRAVIS) {
-    grunt.task.run('shell:npm-install');
```

Comment:
FWIW, I think this was kind of useful...
<sub>(I can leave without it though.)</sub>